client/my-sites/checkout/checkout/payment-chat-button.jsx

```diff
@@ -4,20 +4,30 @@
  * External dependencies
  */
 
-import React from 'react';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
 import Gridicon from 'gridicons';
 
 /**
  * Internal dependencies
  */
 import HappychatButton from 'components/happychat/button';
+import { recordTracksEvent } from 'state/analytics/actions';
 
-export default localize( ( { translate } ) => {
-	return (
-		<HappychatButton className="checkout__payment-chat-button">
-			<Gridicon icon="chat" className="checkout__payment-chat-button-icon" />
-			{ translate( 'Need help? Chat with us' ) }
-		</HappychatButton>
-	);
-} );
+export class PaymentChatButton extends Component {
+	chatButtonClicked = () => {
+		this.props.recordTracksEvent( 'calypso_presales_chat_click' );
+	};
+
+	render() {
+		return (
```

Comment:
For the sake of consistency with other components and to make the code a bit cleaner we could use destructuring assignment here and have something like `const { translate } = this.props;` and use `translate` instead of `this.props.translate`